client/blocks/login/error-notice.jsx

```diff
@@ -61,6 +61,11 @@ class ErrorNotice extends Component {
 			return null;
 		}
 
+		// The existing_wpcom_user error is handled inline in LoginForm
```

Comment:
Apologies if I'm missing something but this comment looks inaccurate to me - I don't see anything in `<LoginForm />`. If we indeed need to display this error below the `Username or Email Address` field, we can specify that [here](https://github.com/Automattic/wp-calypso/blob/85e9ddee8ac63b31514f8756d5bb31d19e1813a1/client/state/login/actions.js#L69-L77).